server.js

```diff
@@ -1,15 +1,17 @@
-var path = require('path');
-var express = require('express');
+// const path = require('path');
+// const express = require('express');
+import path from 'path';
+import express from 'express';
 
-var app = express();
+const app = express();
 
 app.use(express.static(path.join(__dirname, '/dist')));
 
-app.get('*', function(req, res){
-  res.sendfile("index.html", {root: path.join(__dirname, '/dist')});
+app.get('*', (req, res) => {
+  res.sendfile('index.html', { root: path.join(__dirname, '/dist') });
```

Comment:
Hi @idmega2000. `res.sendfile` is deprected. Please use `res.sendFile` instead. Also after working with @Danielshow yesterday we found out that the `path` might not be correct. Instead use either `path.join(__dirname, 'index.html')` or `path.resolve(__dirname, 'index.html')`.